addons/spreadsheet/static/src/helpers/constants.js

```diff
@@ -5,9 +5,16 @@ import { _t } from "@web/core/l10n/translation";
 
 export const DEFAULT_LINES_NUMBER = 20;
 
-export const HEADER_STYLE = { fillColor: "#E6F2F3" };
-export const TOP_LEVEL_STYLE = { bold: true, fillColor: "#E6F2F3" };
```

Comment:
this variable is still used in list_core_plugin, it should not be removed.

Reply:
we probably should also change list to be Tables now that I think about it

Reply:
I think we can leave it for later, not a 100% sure of what result we want... We should discuss this with @Francois-frgi